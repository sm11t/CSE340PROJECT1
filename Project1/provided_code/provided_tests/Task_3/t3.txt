TASKS
    1 3 2
POLY
    F1 = x + 1;
    F2(X,Y) = X^2 Y^2;
    G1(X,Y,Z) = 3 + (X + Y) + 2 (Y + 3Z);
EXECUTE
    INPUT K;
    INPUT L;
    w = F1(K);
    OUTPUT w;
    ww = F2(
        L,
        F1(
            Q
        )
    );
    OUTPUT ww;
    www = G1(
        K,J,

        F2(3,
        
            F1(M)
        )
    );

INPUTS
    3 5 6
