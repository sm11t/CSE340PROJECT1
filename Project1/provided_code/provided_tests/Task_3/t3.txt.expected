4
25
Warning Code 1: 15 20 24